graphics/MoltenVK/Portfile

```diff
@@ -21,52 +21,72 @@ long_description    ${name} is an implementation of the high-performance, \
                     industry-standard Vulkan graphics and compute API, that \
                     runs on Apple's Metal graphics framework.
 
-distname            vulkansdk-macos-${sdkversion}
-use_dmg             yes
+if {${os.major} == 16} {
```

Comment:
> Dean, can you clarify what needs a bump?

This section here would need to be bumped _after_ `MoltenVK-1.10.1_0.darwin_16.x86_64` package is avaliable from packages.macports

Reply:
> This section here would need to be bumped _after_ `MoltenVK-1.10.1_0.darwin_16.x86_64` package is avaliable from packages.macports

What would it need to be changed to? (Sorry for being dense, but I'm not in-tune with all of the details...)

Reply:
> What would it need to be changed to? (Sorry for being dense, but I'm not in-tune with all of the details...)

After the packages are build this section would need to be bumped to use the 1.10.0 so update `version` here and the checksums to match the prebuild 1.10.0 darwin16 package once it's available for download from packages.macports.

```
if {${os.major} == 16} {
    version             1.1.9
    master_sites        https://packages.macports.org/MoltenVK/
    distname            MoltenVK-${version}_0.darwin_16.x86_64
    checksums           sha256  84c49e85712d2bb19f5d00bd91df4dae9518bc31ca3265c56c1fdabae25fcf08 \
                        rmd160  bf7e89fc3f67c0aeb392149eae4ce51ee06d4f06 \
                        size    7026887
    use_bzip2           yes
    extract.suffix      .tbz2
```

Reply:
> After the packages are build this section would need to be bumped to use the 1.10.0 so update `version` here and the checksums to match the prebuild 1.10.0 darwin16 package once it's available for download from packages.macports.

Could this be accomplished via a bootstrap-ish subport, which the parent depends on?

Reply:
> Could this be accomplished via a bootstrap-ish subport, which the parent depends on?

All this port does is unpack the LunarG VulkanSDK and grab the files, the _workaround_ does exactly what a bootstrap port would accomplish.

Reply:
Because the workaround is for 10.11 it should instead say:
```suggestion
if {${os.major} == 15} {
```

A bump (of what exactly—the revision?) doesn’t seem necessary, the 10.11 builds will just fail until the 10.12 build has been published. I would think it would be preferable to instead hold `version` back on 10.11 until it can be updated. But whatever the approach entails I think others might still find it confusing.

Reply:
> Because the workaround is for 10.11 it should instead say

Done

> A bump (of what exactly—the revision?) doesn’t seem necessary, the 10.11 builds will just fail until the 10.12 build has been published. I would think it would be preferable to instead hold version back on 10.11 until it can be updated withot anything failing. But whatever the approach entails I think others might still find it confusing.

After the MoltenVK-1.10.0 packages are built the workaround section could then be updated to 1.10.0 to match the rest.